postgis/docker-compose.yaml

```diff
@@ -1,12 +1,24 @@
-version: "3.8"
-
+version: '3.8'
 services:
-  postgres:
-    image: postgis/postgis:13-master
-    container_name: local_postgres
-    volumes:
-      - ./data:/var/lib/postgresql/data
-    env_file:
-      - postgres.env
+  db:
+    image: postgres:14.1-alpine
+    restart: always
+    environment:
```

Comment:
it is better reading environment variables from file with `env_file` parameter than writing them one by one to docker compose